### DISCUSSION TITLE
weaather.gov > Tasker

### DISCUSSION THREAD

**Discussion by lunis-j** [weather-gov/api#757] [Q&A]
Hope this is the proper place.
I have the proper weather.com API URL for current data.  I have the proper header.  Entered both in to Tasker.  I'm unable to call weather data (e.g., humidity).  How do I make data calls?

**Comment by dwhitemv25** [comment]
What was the URL you were querying?

We've seen problems with API query engines getting blocked by the CDN as being bots or making too many queries too fast. If the same URL works in a Web browser, then its likely the service is being blocked.

**Comment by lunis-j** [comment]
As I understand weather.com and openweathermap.org (I'm trying both), subscribers are allowed 500 or 1000 data calls/mo.  No limit on frequency.  Even if frequency is limited, researching, editing, and attempting a call can be 5 min to 30 min.  I can call http_data and get current weather data, so I'm guessing, the call gets to and is responded by the providers.  And both services claim I'll be contacted if I abuse the call limits.  Finally, if the service is being blocked, is it permanent or for, say, one hr?  I can not make a call for two days, but on trying again, I can retrieve no data, sans http_data.
Both weather services allow me to call and save http_data.  Since I can call http_data file, is there a way to extract data (e.g., pressure) from that file?

**Comment by lunis-j** [comment]
Update.
I've now tried weather.gov, openweathermap, tomorrow.oi, surrise-sunset.org.  None of them work.  All run in Tasker without error, but none display the actual value.  For example, calling '%pressure' returns '%pressure.'  That's true in "Flash" and "Send to WM."  I found an app called Autoweb.  Tried setting up openweathermap and Google elevation.  Neither work.
Is it possible I'm missing something in the setup?  I have a hard time believing none of those APIs work and all have the same problem.

**Reply by lunis-j** [reply]
Any suggestions?

**Reply by dwhitemv25** [reply]
Seems like your issue is with Tasker and not the API. Have you tried getting help from other Tasker users?
